### PR TITLE
[PW-7319] Configuration API Services

### PR DESCRIPTION
**Pull request by jillingk**
**Description**
This PR adds Configuration API services based on the service generator. 

**Tested scenarios**
UT's

### CODE REVIEW

File: Adyen/Constants/ClientConfig.cs [hunk 1]
```diff
@@ -59,6 +59,9 @@ public class ClientConfig
         public static string LegalEntityManagementEndpointTest = "https://kyc-test.adyen.com/lem/";
         public static string LegalEntityManagementEndpointLive = "https://kyc-live.adyen.com/lem/";
         public static string LegalEntityManagementVersion = "v2";
+        public static string BalancePlatformEndpointTest = "https://balanceplatform-api-test.adyen.com/bcl/";
+        public static string BalancePlatformEnndpointLive = "https://balanceplatform-api-test.adyen.com/bcl/";
```
Comment (Kwok-he-Chu) [inline Adyen/Constants/ClientConfig.cs:63]: Suggestion: rename to `BalancePlatformEnndpointLive` to `BalancePlatformEndpointLive` and point url to '-live' 👍

Comment (jillingk) [reply Adyen/Constants/ClientConfig.cs:63]: wow stupid mistake my bad

File: Adyen/Service/Resource/BalancePlatformResource.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#region License
+// /*
```
Comment (wboereboom) [inline Adyen/Service/Resource/BalancePlatformResource.cs:2]: let's lose the ascii banner to save the planet

File: Adyen/Model/BalancePlatform/Address2.cs [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+using Newtonsoft.Json.Linq;
+using System.ComponentModel.DataAnnotations;
+
+namespace Adyen.Model.BalancePlatform
+{
+    /// <summary>
+    /// Address2
+    /// </summary>
+    [DataContract(Name = "Address-2")]
+    public partial class Address2 : IEquatable<Address2>, IValidatableObject
```
Comment (michaelpaul) [inline Adyen/Model/BalancePlatform/Address2.cs:34]: Can we avoid this "-2" model?

Comment (wboereboom) [reply Adyen/Model/BalancePlatform/Address2.cs:34]: The devs need to go back and rename the model. it's not a breaking change for the API. we just need to know where & what to change

**Review by AlexandrosMor — APPROVED** [review]

File: Adyen/Constants/ClientConfig.cs [hunk 4]
```diff
@@ -61,6 +61,9 @@ public class ClientConfig
         public static string StoredValueEndpointTest = "https://pal-test.adyen.com/pal/servlet/StoredValue";
         public static string StoredValueEndpointLive = "https://pal-live.adyen.com/pal/servlet/StoredValue";
         public static string StoredValueVersion = "v46";
+        public static string BalancePlatformEndpointTest = "https://balanceplatform-api-test.adyen.com/bcl/";
+        public static string BalancePlatformEndpointLive = "https://balanceplatform-api-live.adyen.com/bcl/";
+        public static string BalancePlatformVersion = "v2";
```
Comment (antolo-arch) [inline Adyen/Constants/ClientConfig.cs:66]: are we using v2 or v1 to use the x-methodName?

Comment (jillingk) [reply Adyen/Constants/ClientConfig.cs:66]: We can't use v1, it's completely messed up.

File: Adyen/Model/BalancePlatform/Address2.cs
Comment (wboereboom) [inline Adyen/Model/BalancePlatform/Address2.cs:1]: oh no :(

Comment (jillingk) [reply Adyen/Model/BalancePlatform/Address2.cs:1]: Damn..

**Review by michaelpaul — APPROVED** [review]

File: Adyen/Model/BalancePlatform/Address2.cs
Comment (jillingk) [reply Adyen/Model/BalancePlatform/Address2.cs:1]: @wboereboom there was somebody looking into this address2 class right or?

Comment (jillingk) [reply Adyen/Model/BalancePlatform/Address2.cs:1]: @wboereboom pls :(

File: templates/csharp/model.mustache [hunk 5]
```diff
@@ -46,4 +46,4 @@ namespace {{packageName}}.{{modelPackage}}
 {{#isEnum}}{{>modelEnum}}{{/isEnum}}{{^isEnum}}{{#oneOf}}{{#-first}}{{>modelOneOf}}{{/-first}}{{/oneOf}}{{#anyOf}}{{#-first}}{{>modelAnyOf}}{{/-first}}{{/anyOf}}{{^oneOf}}{{^anyOf}}{{>modelGeneric}}{{/anyOf}}{{/oneOf}}{{/isEnum}}
 {{/model}}
 {{/models}}
-}
+}
```
Comment (antolo-arch) [inline templates/csharp/model.mustache:49]: Add the blank line at the end of file again :D

**Review by michaelpaul — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]

**Pull request merged**